File: tests/fixtures/repo-spring/src/main/java/com/example/ReportController.java
package com.example;

@RestController
public class ReportController {
    @PostMapping("/api/report")
    public Map<String, Object> report(@RequestParam String email) {
        jdbcTemplate.execute("select * from users where email = '" + email + "'");
        return Map.of("ok", true);
    }
}

File: tests/fixtures/repo-aspnet/Controllers/ReportController.cs
namespace Example.Controllers;

[ApiController]
[Route("api/report")]
public class ReportController : ControllerBase
{
    [HttpPost]
    public IActionResult Post(string email) {
        _db.ExecuteSql("select * from Users where Email = '" + email + "'");
        return Ok();
    }
}
